contrib/gonum/plotter.go

```diff
@@ -0,0 +1,155 @@
+package gonum
```

Comment:
not completely sure calling this package `gonum` is the best move.
it's more about plotting, right?

Reply:
qplot, or qfplot ?

Reply:
Brilliant! I went with `qfplot`.